tests/test_web/test_api_v1.py

```diff
@@ -123,51 +123,103 @@ def test_get_builds_invalid_state(app, client, db):
         ),
-        (['some:thing'], 32, 'binary:image', ['s390x'], '"from_index" must be a string'),
         (
-            ['something'],
-            'pull_spec',
-            'binary_image',
-            [1, 2, 3],
+            {
+                'bundles': ['some:thing'],
+                'from_index': 32,
+                'binary_image': 'binary:image',
+                'add_arches': ['s390x'],
+            },
+            '"from_index" must be a string',
+        ),
+        (
+            {
+                'bundles': ['some:thing'],
+                'from_index': 'pull:spec',
+                'binary_image': 'binary:image',
+                'add_arches': [1, 2, 3],
+            },
             'Architectures should be specified as a non-empty array of strings',
         ),
     ),
 )
-def test_add_bundle_invalid_params_format(
-    bundles, from_index, binary_image, add_arches, error_msg, db, auth_env, client
-):
-    data = {
-        'bundles': bundles,
-        'from_index': from_index,
-        'binary_image': binary_image,
-        'add_arches': add_arches,
-    }
-
-    rv = client.post('/api/v1/builds/add', json=data, environ_base=auth_env)
+def test_add_bundles_invalid_params_format(data, error_msg, db, auth_env, client):
+    rv = client.post(f'/api/v1/builds/add', json=data, environ_base=auth_env)
     assert rv.status_code == 400
     assert error_msg == rv.json['error']
 
 
-def test_add_bundle_missing_required_param(db, auth_env, client):
+def test_rm_operators_invalid_params_format(db, auth_env, client):
     data = {
-        'from_index': 'from_index',
+        'from_index': 'pull:spec',
         'binary_image': 'binary:image',
-        'add_arches': ['add_arches'],
+        'add_arches': ['s390x'],
     }
+    rv = client.post(f'/api/v1/builds/rm', json=data, environ_base=auth_env)
+    assert rv.status_code == 400
+    assert '"operators" should be a non-empty array of strings' == rv.json['error']
 
-    rv = client.post('/api/v1/builds/add', json=data, environ_base=auth_env)
+
+@pytest.mark.parametrize(
+    'data, error_msg',
+    (
+        (
+            {'from_index': 'pull:spec', 'binary_image': 'binary:image', 'add_arches': ['s390x']},
+            '"bundles" should be a non-empty array of strings',
+        ),
+        (
+            {'bundles': ['some:thing'], 'from_index': 'pull:spec', 'add_arches': ['s390x']},
+            'Missing required parameter(s): binary_image',
+        ),
+    ),
+)
+def test_add_bundle_missing_required_param(data, error_msg, db, auth_env, client):
+    rv = client.post(f'/api/v1/builds/add', json=data, environ_base=auth_env)
     assert rv.status_code == 400
-    assert rv.json['error'] == 'Missing required parameter(s): bundles'
+    assert rv.json['error'] == error_msg
+
+
+@pytest.mark.parametrize(
+    'data, error_msg',
+    (
+        (
+            {'operators': ['some:thing'], 'from_index': 'pull:spec', 'add_arches': ['s390x1']},
```

Comment:
Could you also fix these?

```suggestion
            {'operators': ['some:thing'], 'from_index': 'pull:spec', 'add_arches': ['s390x']},
```

Reply:
Lol, looks like my typo spread wide..sure I'll fix it